src/renderer/services/httpClient.ts

```
@@ -0,0 +1,51 @@
+import axios, { AxiosError, AxiosInstance, AxiosResponse } from 'axios';
+
+export class HttpClient {
+    private client: AxiosInstance;
+
+    public constructor(opt: Partial<string>) {
+        this.client = Axios.create({
+            baseURL: opt || ''
```

I think you should throw error if url is missing